docs/contributing/building/nocore-ebpf.md

```diff
@@ -130,11 +133,14 @@ TIME             UID    COMM             PID     TID     RET              EVENT
 ...
 ```
 
-One way of forcing **tracee-ebpf** to use non CO-RE eBPF object, even in a kernel
+One way of forcing **tracee** to use non CO-RE eBPF object, even in a kernel
 that supports CO-RE, is by setting the `TRACEE_BPF_FILE` environment, like this:
 
+```console
+sudo TRACEE_BPF_FILE=/tmp/tracee/tracee.bpf.5_4_0-91-generic.v0_8_0-rc-2-363-g3e73eeb1.o ./dist/tracee --log debug -o option:parse-arguments --filter comm=bash --filter follow
 ```
-$ sudo TRACEE_BPF_FILE=/tmp/tracee/tracee.bpf.5_4_0-91-generic.v0_8_0-rc-2-363-g3e73eeb1.o ./dist/tracee-ebpf --log debug -o option:parse-arguments --filter comm=bash --filter follow
+
+```text
```

Comment:
json?

Reply:
Not in this case since what we really expect is the table output.

```
...
{"level":"debug","ts":1670972956.7216172,"msg":"BPF: using BPF object from environment","file":"/tmp/tracee/tracee.bpf.5_4_0-91-generic.v0_8_0-rc-2-363-g3e73eeb1.o","pkg":"initialize","file":"bpfobject.go","line":52}
TIME             UID    COMM             PID     TID     RET              EVENT                ARGS
...
```